templates/jobs/drupal-phpqa.yml

```diff
@@ -15,7 +15,7 @@
 .drupal-phpqa-template:
   before_script:
     - ""
-  image: ghcr.io/sparkfabrik/drupal-qa:latest
+  image: ghcr.io/sparkfabrik/drupal-qa:1.0.0
```

Comment:
we can make this one a variable (default to 1.0.0), so it will be easier to control the version from a project/pkg

Reply:
it should be possible: https://docs.gitlab.com/ee/ci/yaml/includes.html#override-included-configuration-values - variables defined outside override the one defined in the included template.

Reply:
I've defined a new variable: `PKG_DRUPAL_QA_VERSION` to be 1.0.0 and I set the image to use that value. This should be enough to let this work as the current version.

Reply:
Thanks